src/forms/checkbox.test.jsx

```diff
@@ -81,7 +81,7 @@ describe('Checkbox', function() {
 			expect(fakeOnChange).not.toHaveBeenCalled();
 			checkbox.simulate('change', { target: { checked: true } });
 			expect(checkbox.props().checked).toBe(true);
-			expect(fakeOnChange).toHaveBeenCalledWith(true);
+			expect(fakeOnChange).toHaveBeenCalled();
```

Comment:
If possible, let's use `toHaveBeenCalledWith` here still by making `{ target: { checked: true } }` a variable.

Reply:
ok. I'm hoping that would work or if I have to ass other properties to fill out the fake event (`preventDefault`, `currentTarget` etc)